src/lib/validations/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * We have to use named exports from this file, otherwise the following
+ * runtime error occurs:
+ *
+ * Uncaught TypeError: Super expression must either be
+ * null or a function, not undefined
+ */
```

Comment:
I would prefer importing the classes directly, but I couldn't resolve the weird runtime problem. TypeScript and Webpack had no issues with the code, though 🤔